healthcheck/healthcheck_test.go

```diff
@@ -227,28 +232,34 @@ func TestNewRequest(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		backend := NewBackendHealthCheck(
-			Options{
-				Path: test.path,
-				Port: test.port,
-			})
+		test := test
+		t.Run(test.desc, func(t *testing.T) {
+			t.Parallel()
+			backend := NewBackendHealthCheck(
+				Options{
+					Path: test.path,
+					Port: test.port,
+				})
 
-		u := &url.URL{
-			Scheme: "http",
-			Host:   test.host,
-		}
+			u := &url.URL{
+				Scheme: "http",
+				Host:   test.host,
+			}
 
-		req, err := backend.newRequest(u)
+			testCase := fmt.Sprintf("URL=%s/%s port=%d", u, test.path, test.port)
```

Comment:
Now that you've added a description to each test case and injected that into the `t.Run()` call, we don't need the prefix anymore. :-)

Would you mind removing just that part again?